src/js/login.js

```diff
@@ -0,0 +1,20 @@
+import $ from 'jquery';

```

Comment:
It's very dangerous using jquery along with react. We already know you can do jquery, I believe the exercise is to do it in react.